jimpass/managers/bitwarden.py

```diff
@@ -1,4 +1,4 @@
-from jimpass.util import srun, rofi
+from jimpass.util import srun, rofi, wofi
```

Comment:
see comment in `util.py`. merge these two functions into one and add the `runner` argument.